code/game/objects/items/weapons/improvised_components.dm

```diff
@@ -144,4 +144,42 @@
 	icon_state = "buckler1"
 	force_divisor = 0.1
 	thrown_force_divisor = 0.1
-	default_material = "steel"
\ No newline at end of file
+	default_material = "steel"
+
+/obj/item/woodcirclet
+	name = "wood circlet"
+	desc = "A small wood circlet for making a flower crown."
+	icon = 'icons/obj/buildingobject.dmi'
+	icon_state = "woodcirclet"
+	item_state = "woodcirclet"
+	w_class = ITEMSIZE_SMALL
+
+/obj/item/woodcirclet/attackby(obj/item/W as obj, mob/user as mob)
+	var/obj/item/complete
+	if(istype(W,/obj/item/seeds/poppyseed))
+		to_chat(user,"You attach the poppy to the circlet and create a beautiful flower crown.")

```

Comment:
Uh not sure how to_chat handles this but generally you should put a span("notice",[text]) so it has a consistent font color. This issue exists through most to_chat calls.

Reply:
Yeah, `to_chat()` doesn't does same thing as `user <<`, so it won't put span by itself, so this will be a plain text.

Reply:
this has been fixed today already, poze